CONTRIBUTING.md

```diff
@@ -1,45 +1,93 @@
 
-### Coding Style
-Before submitting a pull request, make sure the coding style is qualified. run command `pylint --rcfile=pylint.conf milvus/client` 
-to check it.
 
+## What contributions can you make?
 
-## Run unit test with code coverage
+If you identify any problems, you can:
+- file an issue to point out the problem
+- provide minimal reproducible steps in the issue (optional)
+- provide solutions to the issue (optional)
+- create a PR to address this issue (optional)
 
-Before submitting your PR, make sure you have run unit test, and your code coverage rate is >= 90%.
+If you are interested in existing problems, you can:
+- in issues with `question` labels, answer questions from others to offer help
+- in issues with `bug`, `improvement`, `enhancement`:
+  - question, reproduce, or provide solutions
+  - create a PR to repair the issue
 
-```shell 
-$ pytest --cov=milvus/client --cov-report=html
-```
+If you want to request more features for PyMilvus, you can:
+- file an issue to describe the new features and explain why
+- provide implementation scheme and test scheme (optional)
+- create a PR to realize the new features (optional)
 
-You may need a milvus server which is running when you run unit test. See more details on [Milvus server](https://github.com/milvus-io/milvus).
+If you are interested in existing PRs, you can:
+- join the code review and offer advice
+- guide newcomers to finish the PR process
 
+Note: the problems, features, and questions mentioned here are not limited to Python code. They also refer to all kinds of documents (technical documents, API references, contributing guide, etc.)
 
-## Update CHANGLOG.md
+## PyMilvus Code Structure
+`docs/`: Contains source codes for documentation except for API Reference and uses sphinx to create documentation. Most documentation are in folder `rst` under `docs/source`.
 
-Add issue tips into CHANGLOG.md, make sure all issue tips are sorted by issue number in ascending order.
+`examples/`: Contains python scripts, which can be run directly, to introduce the usage of PyMiluvs API through examples.
 
+`pymilvus/`：PyMilvus source codes.
 
+`tests/`: Unit Test.
 
+`CONTRIBUTING.md`: Contributing guidelines.
+
+`CONTRIBUTING_CN.md`: Contributing guidelines in Chinese.
+
+`LICENSE`: Open Source License that PyMilvus follows.
+
+`Makefile`: Scripts for Github action.
+
+`OWNERS`: This file designates reviewers and approvers for the current directory. They are chosen according to their activity and code contribution. Active contributors are listed as reviewers, undertaking the responsibility of code reviews. Reviewers who have been active and reviewing codes for a period of time are listed as approvers. They are in charge of reviewing content apart from codes. If you create a PR and do not know who to review, you can assign reviewers and approvers from this file to review your PR.
```

Comment:
`OWNERS`: This file designates reviewers and approvers for the current directory. They are chosen according to their participation and code contribution. Active contributors are listed as reviewers, responsible for code reviews. Reviewers who have been active and reviewing codes for a period of time are listed as approvers. They are in charge of reviewing content apart from codes. If you submit a PR and do not know who can help you review the code, you can assign reviewers and approvers from this file to review your PR.